### PR TITLE
Module7 ci

### PR DESCRIPTION
**Pull request by LouisLon**

### CODE REVIEW

File: todo_app/tests_e2e/app_e2e_test.py
Comment (JackMeadDev) [inline todo_app/tests_e2e/app_e2e_test.py]: If we're switching to chromedriver, let's delete this

File: Dockerfile
Comment (JackMeadDev) [inline Dockerfile]: We shouldn't need these environment variables, since our tests define any relevant config

File: Dockerfile
Comment (JackMeadDev) [inline Dockerfile]: Similarly, this test isn't expecting to expose anything outside the container, so we don't need this line

**Review by JackMeadDev — APPROVED** [review]
Nice, I can see your tests running & passing on travis-ci.org - it's worth noting that that is being sunsetted in favour of travis-ci.com, where your tests are also currently running but [failing](https://travis-ci.com/github/LouisLon/DevOps-Course-Starter/builds/219020431). Looks like that's because you haven't set the environment variables there, so before merging, you should add the relevant variables on that site too, and I'd suggest disabling travis-ci.org once that's working so that we have one source of truth for the tests.

Otherwise, I've left a few thoughts below, but happy for you to merge this when you're ready

**Comment by LouisLon** [comment]
Count not run on https://travis-ci.com/ as it does not pick up the LouisLon/DevOps-Course-Starter repository

**Pull request merged**
